test/gtest/uct/test_peer_failure.cc

```diff
@@ -314,10 +365,112 @@ UCS_TEST_P(test_uct_peer_failure_cb, desproy_ep_cb)
     check_caps(UCT_IFACE_FLAG_PUT_SHORT);
 
     wrap_errors();
-    kill_receiver0();
+    kill_receiver();
     EXPECT_EQ(uct_ep_put_short(ep0(), NULL, 0, 0, 0), UCS_OK);
     flush();
     restore_errors();
 }
 
 UCT_INSTANTIATE_TEST_CASE(test_uct_peer_failure_cb)
+
+class test_uct_peer_failure_multiple : public test_uct_peer_failure
+{
+public:
+    virtual void init();
+
+protected:
+    size_t get_tx_queue_len() const;
+};
+
+void test_uct_peer_failure_multiple::init()
+{
+    if (RUNNING_ON_VALGRIND) {
+        /* See https://bugs.kde.org/show_bug.cgi?id=352742 */
+        UCS_TEST_SKIP_R("skipping on valgrind because \"brk segment overflow\"");
+    }
+
+    size_t tx_queue_len = get_tx_queue_len();
+
+#if defined(__arm__) || defined(__aarch64__)
```

Comment:
do we still need that?
better check that page_size()>4k because this is relevant also for ppc64